docs/proposal/bypass.md

```diff
@@ -0,0 +1,72 @@
+## Background
+
+Currently, if the mesh service is turned on when communication between servers fails, it is difficult to determine whether the problem is caused by the mesh service taking over the traffic, or whether there is a logic error in the application itself. Therefore, we provide a feature to shut down the mesh service and remove it from the communication traffic path to pinpoint whether the problem is caused by the mesh service or an error in the application itself.
+
+## Bypass Function Design
+
+Bypass is a general mesh function used to remove sidecar/kmesh mesh services from the traffic path. The following two scenes show the traffic path after the bypass function is enabled.
+
+### Scene 1: sidecar
+
+Among these two figures, the mesh service is enabled in the cluster to take over the traffic. The first figure shows the traffic path before the bypass function is enabled. We can see from it that the traffic path is server A -> sidecar -> sidecar - > server B, when the communication between server A and server B fails, from the black box level, the user cannot determine whether there is a problem with the sidecar or the server itself. In Figure 2, the bypass function of pod1 is enabled, so that the sidecar in pod1 is removed from the traffic path. If server A and server B can communicate normally at this time, it proves that there is a problem with the sidecar.
+
+![alt text](pics/sidecar_pre_bypass.svg)
+
+![alt text](pics/sidecar_bypass.svg)
+
+### Scene 2: kmesh
+
+![alt text](pics/kmesh_pre_bypass.svg)
+
+![alt text](pics/kmesh_bypass.svg)
+
+## Bypass fields
+
+```c
+struct bpf_map_def SEC("maps") bypass_fliter_map = {
+    .type = BPF_MAP_TYPE_HASH,
+    .key_size = sizeof(__u32),
+    .value_size = sizeof(__u8),
+    .max_entries = MAX_ENTRIES,
+};
+```
+
+The key represents the IP address of the pod, and the value represents whether the bypass function is enabled for the current pod, represented by 0/1. When this ebpf program is executed, it will check whether the corresponding pod has the bypass function enabled. If enabled, all subsequent iptables rules will be short-circuited, otherwise, subsequent iptables rules will not be short-circuited.
+
+## How to enable bypass
+
+Users can label pods with the following command:
+
+```shell
+kubectl label pod <pod_name> bypass=1
+```
+
+This causes the inbound and outbound traffic of the mesh service on the pod to be short-circuited.
+
+## Traffic governance process
+
+### Scene 1: sidecar
+
+
+![alt text](pics/sidecar_traffic_path.svg)
+
+The path represented by the black arrow in the above figure is the traffic path before the bypass function is enabled, and the blue arrow is the traffic path after the bypass function is enabled. The difference is that after the bypass function is enabled, two will be added to the top layer of the iptables rules by CNI. The rules are:
```

Comment:
>two will be added to the top layer of the iptables rules by CNI

the two rules are inserted in front of the iptables rules set up by CNI.

And which CNI do you mean, be explicitly if this is kmesh mda

Reply:
> > two will be added to the top layer of the iptables rules by CNI
> 
> the two rules are inserted in front of the iptables rules set up by CNI.
> 
> And which CNI do you mean, be explicitly if this is kmesh mda

I've updated the documentation to clearly state that it is mda